ctlearn/default_models/param_model.py

```diff
@@ -0,0 +1,15 @@
+import importlib
+import sys
+import tensorflow as tf
+
+def param_model(data, model_params):
```

Comment:
This looks good already! Once you have time please try to merge this feature in the existing models and make it configurable.